src/redux/actionCreators.js

```diff
@@ -7,6 +7,11 @@ const addPostCreator = (newPostText) => ({
   newPostText,
 });
 
+const deletePost = (postId) => ({
+  type: ACTION_TYPES.DELETE_POST,
+  postId,
```

Comment:
Consider keeping the action object schema as { type, payload }

Reply:
done